supporting_content/columns/amortizationcategory.md

```diff
@@ -0,0 +1,52 @@
+# Column: Amortization Category
+
+Objective: we need a simple method to calculate savings that doesn't accidentally double-count commitment discount purchase costs.
```

Comment:
```suggestion
Objective: we need a simple method to calculate savings that doesn't accidentally double-count amortized purchase costs.
```

Reply:
Monthly fees can still be amortized. Amortization [as I understand it] is the process of taking any charge and splitting it out by the time granularity of the dataset. So if I amortize a charge that has a 7-day applicability period, then I would see 7 amortized rows in a daily dataset and 168 amortized rows in an hourly dataset. Conversely, if I have a monthly dataset, then there'd only be 1 amortized row. All of these are on top of the original/principal/source charge.

Reply:
Agree, per above.